distribution/docs/src/main/resources/_contents/_running/starting-intro-contents.adoc

```diff
@@ -35,12 +35,12 @@ If console access is needed while running as a service, run the `client` script:
 
 .*NIX
 ----
-<${branding}_HOME/bin/client
+<${branding}_HOME>/bin/client
 ----
 
 .Windows
 ----
-<${branding}_HOME>/bin/client.bat
+<${branding}_HOME>/bin/client.bat -h localhost
```

Comment:
It should be fine, since that should work by default. I was wondering if they were on a separate machine if they would have to put in -h `hostname`. If so, we could just put a note to specify the hostname of the host running DDF. For example...